Bio/SeqUtils/__init__.py

```diff
@@ -11,31 +11,136 @@
 
 
 import re
+import warnings
+from collections import Counter
 from math import pi, sin, cos
 
 from Bio.Seq import Seq, complement, complement_rna
 from Bio.Data import IUPACData
-
+from Bio import BiopythonDeprecationWarning
 
 ######################################
 # DNA
 ######################
 # {{{
 
+_ambiguous_gc_values = {
+    "A": 0,
+    "C": 1.0,
+    "T": 0,
+    "G": 1.0,
+    "S": 1.0,
+    "V": 2 / 3,
+    "B": 2 / 3,
+    "M": 0.5,
+    "R": 0.5,
+    "Y": 0.5,
+    "K": 0.5,
+    "X": 0.5,
+    "N": 0.5,
+    "H": 1 / 3,
+    "D": 1 / 3,
+    "W": 0,
+}
+
+
+def gc_content(seq: str, ambiguous: str = "ignore"):
```

Comment:
Can `seq` be a `Seq`, `MutableSeq`, or `SeqRecord`?

Reply:
Yes to the first, and quite possibly a SeqRecord too.

Unless we can get the type annotations tested under continuous integration, my inclination is to leave them out.

Reply:
Will remove type annotations